apps/nextjs-website/react-components/components/Footer/components/FooterColumn.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link, Stack, Typography } from '@mui/material';
+import { hrefNoOp } from '../../../types/common/Common.types';
+import { LogoPagoPACompany } from '../assets/LogoPagoPACompany';
+import { EIcon } from '../../../components/common/EIcon';
+import { FooterColumnProps } from '../../..//types/Footer/Footer.types';
+import { Button } from '@mui/material';
+
+export const FooterColumn = ({
+  data,
+  companyLink,
+  icons,
+}: FooterColumnProps) => (
+  <Stack spacing={2} display='flex' alignItems={{ xs: 'center', sm: 'start' }}>
+    {data?.title && <Typography variant='overline'>{data.title}</Typography>}
+
+    {companyLink && (
+      <Button
+        role='link'
+        aria-label={companyLink.ariaLabel}
+        href={companyLink.href ?? hrefNoOp}
+        sx={{ display: 'inline-flex' }}
+      >
+        <LogoPagoPACompany />
+      </Button>
+    )}
+
+    {icons && (
+      <Stack
+        aria-label='i nostri social'
+        spacing={{ xs: 3, sm: 1.5, lg: 3 }}
+        direction='row'
+        component='ul'
+        alignItems={{ xs: 'center', sm: 'start' }}
+        sx={{ padding: 0, mt: 0.5, listStyle: 'none' }}
+      >
+        {icons?.map(
+          ({ icon, href = hrefNoOp, onClick, ...rest }, i) =>
+            icon && (
+              <li key={i}>
+                <Link href={href}>
+                  <EIcon icon={icon} color='text.primary' {...rest} />
+                </Link>
+              </li>
+            )
+        )}
+      </Stack>
+    )}
+
+    <Stack
+      component='ul'
+      alignItems={{ xs: 'center', sm: 'start' }}
+      textAlign={{ xs: 'center', sm: 'left' }}
+      sx={{ padding: 0, listStyle: 'none' }}
+    >
+      {data?.links.map(({ href = hrefNoOp, label, ariaLabel, onClick }, i) => (
```

Comment:
```suggestion
      {data.links.map(({ href = hrefNoOp, label, ariaLabel, onClick }, i) => (
```
`data` isn't optional